src/messages/messages.ts

```diff
@@ -2,7 +2,7 @@ import { Selection } from "monaco-editor";
 import { Editor } from "../editor/editor";
```

Comment:
do we need to separate the code for "messages" from the code for "highlighting code" into two files? as messages.ts doesn't seem to be right anymore.